schemas/context/browserdetails.schema.json

```diff
@@ -16,13 +16,23 @@
         "xdm:name": {
           "title": "Name",
           "type": "string",
-          "description": "The application or browser version."
+          "description": "The application or browser name."
```

Comment:
should this really be browser or the more generic user-agent?  I would prefer that we replace all reference to browser with user-agent

Reply:
I understand the intent of the request. In the the behavioral data the standard is to denormalize to more usable dimensions because it is costly to interpret user-agents.

Reply:
but this material is relevant to any UA (as per HTML specs) and not just a browser....Yes?  If so, we should talk about it in that context.